### ISSUE TITLE
is this project (C#) alive and fully working with the latest Moltin REST API?

### ISSUE THREAD

**Issue by cchacons** [moltin/csharp-sdk#3]
I'd like to confirm/check if this code is fully working with Moltin REST API and there is full support to keep it working. 

Thanks,
Carlos

**Comment by Michael-Brooks** [comment]
Same here, it would be cool to get a response. 👍

**Comment by treed593** [comment]
It seems the clone at https://github.com/r3plica/csharp-sdk is more active and was mentioned in the Moltin Slack room

**Comment by Michael-Brooks** [comment]
Thank you @treed593 I found that fork after commenting here.

**Comment by cchacons** [comment]
Thanks!

**Comment by r3plica** [comment]
Yeah, and I have just updated it today so that you no longer need to get the access token before querying moltin
